eng/pipelines/branch-cleanup.yml

```diff
@@ -16,6 +16,16 @@ parameters:
     - Azure/azure-sdk-for-net
     - Azure/azure-sdk-for-python
 
+- name: ForkAutomationBranches
+  type: object
+  default:
+    - azure-sdk/azure-powershell
```

Comment:
Are we responsible for this repo?

Reply:
It is part of the sdk automation coming from the specs repo and it uses a fork in our bot account so I feel like we should have it in the list.